src/Orleans.Core/Orleans.Core.csproj

```diff
@@ -19,10 +19,6 @@
     </Content>
   </ItemGroup>
 
-  <ItemGroup>
-    <Compile Remove="Statistics\RuntimeStatisticsGroup.cs" />
```

Comment:
This file looked like a duplication of the perf-counter implementation so deleted it